### PR TITLE
DOPS-664 add docker-compose file

### PR DESCRIPTION
**Pull request by himaster**
DOPS-664 add docker-compose file

### CODE REVIEW

**Comment by neonlabstech** [comment]
<details><summary>Solana Requests Statistics</summary>



|                                   |   count |   min_time |   max_time |   average_time |   median_time |
|:----------------------------------|--------:|-----------:|-----------:|---------------:|--------------:|
| getSlot                           |   72922 |      0     |      0.129 |    0.000705274 |         0     |
| getAccountInfo                    |   54166 |      0     |      0.091 |    0.00131848  |         0     |
| getBlocks                         |   27860 |      0     |      0.09  |    0.000469526 |         0     |
| getTransaction                    |   23681 |      0     |      0.42  |    0.00340742  |         0.001 |
| sendTransaction                   |   21646 |      0     |      0.144 |    0.00361411  |         0.002 |
| getBlock                          |   13985 |      0     |      0.039 |    0.00085177  |         0.001 |
| getMultipleAccounts               |    9049 |      0     |      0.102 |    0.00208067  |         0.001 |
| getBlockTime                      |    5793 |      0     |      0.071 |    0.000612981 |         0     |
| getLatestBlockhash                |    4460 |      0     |      0.102 |    0.00194709  |         0.001 |
| getBalance                        |    1755 |      0     |      0.093 |    0.00132023  |         0     |
| getSignatureStatuses              |    1033 |      0     |      0.087 |    0.00109777  |         0     |
| getVersion                        |     538 |      0     |      0.025 |    0.000931227 |         0     |
| isBlockhashValid                  |     498 |      0     |      0.017 |    0.000481928 |         0     |
| getSignaturesForAddress           |     132 |      0     |      0.023 |    0.00225758  |         0.002 |
| simulateTransaction               |      41 |      0     |      0.001 |    0.000219512 |         0     |
| requestAirdrop                    |      32 |      0     |      0.002 |    0.00071875  |         0.001 |
| getHealth                         |       3 |      0     |      0     |    0           |         0     |
| getTokenAccountBalance            |       3 |      0     |      0.001 |    0.000333333 |         0     |
| getFirstAvailableBlock            |       2 |      0     |      0     |    0           |         0     |
| getMinimumBalanceForRentExemption |       1 |      0     |      0     |    0           |         0     |
| getClusterNodes                   |       1 |      0.001 |      0.001 |    0.001       |         0.001 |

</details>

**Review by afalaleev — COMMENTED** [review]
@himaster please rebase your PR, because there were changes in neon-tests image

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    platform: linux/amd64
+    restart: unless-stopped
+    environment:
+      SOLANA_URL: ${SOLANA_URL}
+      POSTGRES_DB: neon-db
+      POSTGRES_USER: neon-db-user
+      POSTGRES_PASSWORD: neon-db-password
+      POSTGRES_HOST: postgres
+      GATHER_STATISTICS: "YES"
+      LOG_FULL_OBJECT_INFO: "YES"
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this variable

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+      CONFIG: custom
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this variable

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+      START_SLOT: LATEST
+      REINDEX_START_SLOT: CONTINUE
+      EVM_LOADER: ${EVM_LOADER:-NeonVMyRX5GbCrsAHnUwx1nYYoJAtskU1bWUo6JGNyG}
+      SLOT_PROCESSING_DELAY: 40
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this variable

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+      POSTGRES_TIMEOUT: 15
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this variable

File: docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+      REINDEX_THREAD_COUNT: 5
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this variable

File: docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+    environment:
+      SOLANA_URL: ${SOLANA_URL}
+      POSTGRES_DB: neon-db
+      POSTGRES_USER: neon-db-user
+      POSTGRES_PASSWORD: neon-db-password
+      POSTGRES_HOST: postgres
+      GATHER_STATISTICS: "YES"
+      LOG_FULL_OBJECT_INFO: "YES"
+      CONFIG: custom
+      START_SLOT: LATEST
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this variable

File: docker-compose.yml [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+      REINDEX_START_SLOT: CONTINUE
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this variable

File: docker-compose.yml [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+      EVM_LOADER: ${EVM_LOADER:-NeonVMyRX5GbCrsAHnUwx1nYYoJAtskU1bWUo6JGNyG}
+      SLOT_PROCESSING_DELAY: 40
+      POSTGRES_TIMEOUT: 15
+      REINDEX_THREAD_COUNT: 5
+    hostname: indexer
+    volumes:
+      - "./id.json:/root/.config/solana/id.json"
+    expose:
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this section

File: docker-compose.yml [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+      - "8887"
+      - "8888"
+    ports:
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this section

File: docker-compose.yml [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+    image: neonlabsorg/neon-proxy.py:${VERSION}
+    platform: linux/amd64
+    restart: unless-stopped
+    environment:
+       SOLANA_URL: ${SOLANA_URL}
+       POSTGRES_DB: neon-db
+       POSTGRES_USER: neon-db-user
+       POSTGRES_PASSWORD: neon-db-password
+       POSTGRES_HOST: postgres
+       LOG_FULL_OBJECT_INFO: "NO"
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this setting

File: docker-compose.yml [hunk 11]
```diff
@@ -0,0 +1,141 @@
...
+       MIN_OPERATOR_BALANCE_TO_WARN: 4565760000 # = 913152000 * 5 (5 storage accounts) = 4.56576 SOL
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this setting

File: docker-compose.yml [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+       MIN_OPERATOR_BALANCE_TO_ERR: 913152000 # = solana rent 131072 (= Rent-exempt minimum: 0.913152 SOL) SOLs to create a storage
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this variable

File: docker-compose.yml [hunk 13]
```diff
@@ -0,0 +1,141 @@
...
+       MINIMAL_GAS_PRICE: 1
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this variable

File: docker-compose.yml [hunk 14]
```diff
@@ -0,0 +1,141 @@
...
+       ENABLE_PRIVATE_API: "NO"
+       ENABLE_SEND_TX_API: "NO"
+       EVM_LOADER: ${EVM_LOADER:-NeonVMyRX5GbCrsAHnUwx1nYYoJAtskU1bWUo6JGNyG}
+       GATHER_STATISTICS: "YES"
+       COMMIT_LEVEL: Confirmed
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this variable

File: docker-compose.yml [hunk 15]
```diff
@@ -0,0 +1,141 @@
...
+       CU_PRIORITY_FEE: 2
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this variable

File: docker-compose.yml [hunk 16]
```diff
@@ -0,0 +1,141 @@
...
+       SIMPLE_CU_PRIORITY_FEE: 1
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this variable

File: docker-compose.yml [hunk 17]
```diff
@@ -0,0 +1,141 @@
...
+       DEBUG_CMD_LINE: "NO"
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this variable

File: docker-compose.yml [hunk 18]
```diff
@@ -0,0 +1,141 @@
...
+       FUZZ_FAIL_PCT: 0
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this variable

File: docker-compose.yml [hunk 19]
```diff
@@ -0,0 +1,141 @@
...
+       CONST_GAS_PRICE: 2
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this variable

File: docker-compose.yml [hunk 20]
```diff
@@ -0,0 +1,141 @@
...
+       MINIMAL_WITHOUT_CHAIN_ID_GAS_PRICE: 1
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this variable

File: docker-compose.yml [hunk 21]
```diff
@@ -0,0 +1,141 @@
...
+       LOG_CLEF_FORMAT: "NO"
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this variable

File: docker-compose.yml [hunk 22]
```diff
@@ -0,0 +1,141 @@
...
+       RUST_BACKTRACE: 0
+       SOLANA_KEY_FOR_EVM_CONFIG: ${SOLANA_KEY_FOR_EVM_CONFIG}
+    hostname: proxy
+    volumes:
+      - "./id.json:/root/.config/solana/id.json"
+    ports:
+      - 127.0.0.1:9090:9090
+      - 127.0.0.1:8889:8888
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this variable

File: docker-compose.yml [hunk 23]
```diff
@@ -0,0 +1,141 @@
...
+    expose:
+      - "8888"
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this variable

File: .gitignore [hunk 24]
```diff
@@ -24,3 +24,6 @@ venv*
 
 dist
 build
+
+indexer_db/
```
Comment (afalaleev) [inline .gitignore]: Not exist directory

File: .gitignore [hunk 25]
```diff
@@ -24,3 +24,6 @@ venv*
...
+id.json
```
Comment (afalaleev) [inline .gitignore]: Not exist file

**Review by afalaleev — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 26]
```diff
@@ -0,0 +1,141 @@
...
+       GATHER_STATISTICS: "YES"
+       COMMIT_LEVEL: Confirmed
+       CU_PRIORITY_FEE: 2
+       SIMPLE_CU_PRIORITY_FEE: 1
+       DEBUG_CMD_LINE: "NO"
+       FUZZ_FAIL_PCT: 0
+       CONST_GAS_PRICE: 2
+       MINIMAL_WITHOUT_CHAIN_ID_GAS_PRICE: 1
+       LOG_CLEF_FORMAT: "NO"
+       RUST_BACKTRACE: 0
```
Comment (afalaleev) [inline docker-compose.yml]: Please remove this variable

**Review by afalaleev — CHANGES_REQUESTED** [review]

**Review by afalaleev — APPROVED** [review]

**Pull request merged**